### PR TITLE
Create get 100,000000 tokens i blooket

### PR DESCRIPTION
**Pull request by Blookethaker**
can you mack it ware we can get 10,000 tokens

### CODE REVIEW

File: get 100,000000 tokens i blooket [hunk 1]
```diff
@@ -0,0 +1 @@
+suck my balls
```
Comment (HJT268) [inline get 100,000000 tokens i blooket:1]: no

**Comment by Blookethaker** [comment]
Hi can you make the add tokens to 10,000 per day

**Review by cortezhayesy — COMMENTED** [review]
thx

**Comment by V9293** [comment]
thx

**Comment by ThirdEyeBoi** [comment]
you cant do that lol

**Pull request closed without merging**